README.md

```diff
@@ -76,28 +76,17 @@ All workflows are using the tagging convention listed below. It is subsequently
 
 ### Get the tools
 
-Download `docker-compose.yml` and `mailserver.env`
-
-``` BASH
-wget https://raw.githubusercontent.com/docker-mailserver/docker-mailserver/master/docker-compose.yml
-wget https://raw.githubusercontent.com/docker-mailserver/docker-mailserver/master/mailserver.env
-```
-
-and the `setup.sh` **in the correct version**
-
 ``` BASH
-# if you're using :edge as the image tag
-wget https://raw.githubusercontent.com/docker-mailserver/docker-mailserver/master/setup.sh
-# if you're using :latest (= :10.1.1) as the image tag
-wget https://raw.githubusercontent.com/docker-mailserver/docker-mailserver/v10.1.2/setup.sh
+DMS_GITHUB_URL='https://raw.githubusercontent.com/docker-mailserver/docker-mailserver/master'
+wget "${DMS_GITHUB_URL}/docker-compose.yml"
+wget "${DMS_GITHUB_URL}/mailserver.env"
+wget "${DMS_GITHUB_URL}/setup.sh"
```

Comment:
Is this last step with the `setup.sh` necessary still?

Can you run `docker exec ...` as an alternative now? (seem to be missing docs example on that if an option).

Is the main intent to still interact with this external script but persist version specific changes internally with releases?

Reply:
Yes, you can run `docker exec .. setup ...` and it should work fine. The mail intend of `setup.sh` in my eyes is to wrap the interaction with the container for many users so they do not have to know the name of the container, etc.

But I agree with the fact that we should note that one could easily run `docker exec ... setup ...` as well.

Reply:
While tackling this, do you think it's worth supporting [this issue request](https://github.com/docker-mailserver/docker-mailserver/issues/2140#issuecomment-902296722) to add a default user when an ENV is set?

Their use case was for CI where they apparently could not easily initialize a user via invoking an explicit script/command. I shared how I added an entry point to automate this for my local testing, we could do similar by adopting an entrypoint with that logic?

If we use an entrypoint in the Dockerfile, then no args can keep the current behaviour, otherwise we could take args provided in a `docker run` and pass them to this internal script now? (I have had some experience with that a while back, [example shared here](https://github.com/gatsbyjs/gatsby-docker/pull/19#issuecomment-449576344))

May be worth it, as that recent blogpost you linked to also mentioned setting up a user as a gripe, having users provide an explicit ENV in their compose file for an admin or dummy user can simplify getting started without needing to get into `setup.sh` as early on?

Reply:
To be honest, I'd rather not like to implement this behaviour in this PR for two reasons:

1. I'm not sure what the solution should look like in the end
2. I'd like to not mix up concerns with PRs. This PR can (and should in my opinion) the base for the suggested changes, but implement them too:)

Reply:
Added the note with a534bf6f02f522ffd229ce1444d08e205988f9de